ideascube/tests/test_models.py

```diff
@@ -34,16 +34,13 @@ def test_create_superuser():
     assert user.is_staff
 
 
-def test_list_users():
+def test_list_users(user, systemuser):
     model = get_user_model()
-    model.objects.create_user('123456')
-    model.objects.create_user('__system__')
-
     users = model.objects.all()
     assert [u.serial for u in users] == ['123456']
 
     users = model.objects.all(include_system_user=True)
-    assert [u.serial for u in users] == ['__system__', '123456']
+    assert set([u.serial for u in users]) == set(['__system__', '123456'])
```

Comment:
Why do you move to a set here?

On the contrary, I'd argue we really want to not use a set and allow for duplicates, because that would be an error which should fail the tests.